message_sender/models.py

```diff
@@ -26,6 +26,7 @@ class Outbound(models.Model):
         "been answered. Not used for text messages")
     attempts = models.IntegerField(default=0)
     metadata = JSONField()
+    channel = models.CharField(null=True, blank=True, max_length=64)
```

Comment:
Yup. When the new message object gets created, then we lookup the channel and attach that. Would also mean that we could respond with an HTTP error if the SBM tries to use a channel that doesn't exist.